src/config/eventloop.rs

```diff
@@ -0,0 +1,26 @@
+use rhai::{plugin::*, Scope};
+use rhai::{Dynamic, EvalAltResult, FnPtr};
```

Comment:
Those imports are unused, is this intentional?

Reply:
No, just copy and paste to much, sorry forgot to clean all the stuff. Should be now fine with the last commit :)